ui/src/containers/VolumePageContent.js

```diff
@@ -114,30 +114,30 @@ const VolumePageContent = (props) => {
   }
 
   const queryStartingTime = volumeStats?.queryStartingTime;
-  const volumeUsed = volumeStats?.volumeUsed?.filter(
+  const volumeUsed = volumeStats?.volumeUsed?.find(
     (vU) => vU.metric.persistentvolumeclaim === PVCName,
-  );
-  const volumeThroughputWrite = volumeStats?.volumeThroughputWrite?.filter(
+  )?.values;
+  const volumeThroughputWrite = volumeStats?.volumeThroughputWrite?.find(
```

Comment:
why the change from `filter` to `find`?

Reply:
Since we add the aggregation in the Prometheus query, we have **only one** candidate in the array. 
So `find` is a little bit more efficient than `filter`. Because once it matches the value in findings, it will not check the remaining values in the array collection.

Reply:
Thanks for the explanation :+1: 
This is the kind of detail than can be explained in a commit message :slightly_smiling_face: